package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nest-raven",
-  "version": "9.0.0",
+  "version": "9.0.1",
```

Comment:
Should be 9.1.0 as its a feature and not a bug fix[1]. But we can fix this later.

[1] https://semver.org/

Reply:
Updated here :) 14d477ec06ac1a7538797bb75b9391d6f3cabf93 thanks for the quick review!